ph5/clients/tests/test_ph5toms.py

```diff
@@ -203,6 +206,35 @@ def test_get_nonrestricted_segments(self):
             station_to_cut_list, restricted_list)
         self.assertEqual(result[0].__dict__, expected.__dict__)
 
+    def test_create_trace(self):
```

Comment:
Separate test_create_trace() and test_mismatch_sample_rate to a different class: TestPH5toMSeed_samplerate(LogTestCase, TempDirTestCase) to use tearDown to delete self.ph5_object (use self.ph5_object instead of ph5API_object, self.ph5_sr to utilize tearDown())
LogTestCase help prevent log information to be printed to terminal in unittest.
TempDirTestCase create self.home is the root directory of the project, self.tmpdir is the temporary directory that is supposed to keep all files created during unittest. self.tmpdir will be remove with all in it when each test is finished if the test inherit tearDown from this class. The current directory will be self.tmpdir at the beginning of the test..